### ISSUE TITLE
Find a Form audit

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#17368]
## Description
Complete a full accessibility audit of the Find a Form product, including:
- [Landing page](https://www.va.gov/find-forms/)
- [Search results](https://www.va.gov/find-forms/?q=10-10)
- [Detail pages](https://www.va.gov/find-forms/about-form-10-10065/)

## Acceptance Criteria

- [x] Accessibility audit has been completed
- [x] Recommended updates and improvements have been documented
- [x] Any high priority/critical updates are brought to the attention of PM/DM/PO
- [x] Design review ticket is created

**Comment by laflannery** [comment]
### Accessibility Updates

- Frequently used forms:
  - There is an incorrect aria-label prop on the VA link that needs to be removed because:
    - It doesn’t work on the component and
    - It isn’t needed - the links themselves should be unique
      - If we can’t make these unique we can rethink this
- Consider making  “Search for a VA form by keyword, form name, or form number.” a heading
  - It looks like one but isn’t so if we don’t want it to be, we should consider changing the style
- Form detail pages intro section:
  - The form name, related to and last updated is within a <dl> markup. This is unnecessary for this information.
    - If this truly should be within a list markup, a more traditional <ul> should be used
  - There is also screen reader only text added - “Form name”. However, if this text needs to be defined as the form name for SR users, then we need to define it for all users and make this visible.
- Download modal - We need to determine/codify the download modal behavior
  - These currently open a modal but why? Why wouldn’t this be the case site wide wherever a download form element is used?
  - It is is opening a modal, It should be a button on both the listing and detail so we should go back to the DST and propose a download button style (rather than using the link)
  - If it should not open a modal we should use the DST download link component for the listing and detail page

### Updates that should be made to align with upcoming DST changes:
**Upcoming Heading/Link guidance**
No timeline yet for when [guidance](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2146) will be formalized
- There are quite a few heading links that should potentially get rethought with this new guidance.

**Components**
- [Card](https://design.va.gov/components/card)
  - The frequently used forms blocks should be updated to use the card component

### Other updates/improvements

- Under “Manage your VA benefits and other information online” and related links components the headings have smaller heading styling. 
  - Ultimately these might be reevaluated based on heading/link guidance but if they remain, they should have the default DS styling.
  - This applies to the Headings of the sections themselves and the heading/links within the sections.
  - If this many headings have different styling, there is a larger design issue
- On the search results, the titles of the forms have a font size override
  - Unclear why but I feel like we should remove it regardless
- Search component
  - Ask DS team to consider visible label for search field
